pom.xml

```diff
@@ -121,7 +121,7 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<configuration>
-					<argLine>${surefireArgLine} -Xmx512m</argLine>
+					<argLine>${surefireArgLine} -Xmx1024m</argLine>
```

Comment:
Do you need to increase this when `spring.test.context.cache.maxSize` is set (decreased)?

Reply:
This is set in gitlab, in the values.yaml. So I don't think this is needed at all, except if you are running it locally i guess.

Reply:
This is the sureFireArgLine (settings used during surefire-execution).
The value in values.yaml corresponds to "app.jvm.xmx.mb" in this pom.xml (can also be removed).